test/blackbox/logging/log_receiver.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/ash
```

Comment:
is this a typo or do you really want to use `ash`?  It's probably just a symlink to `sh` anyway on most systems these days

Reply:
Just that I've always tended to use /bin/bash by default and similarly then started using ash on alpine. Will change to sh - don't think I'm using any shell-specific features

Reply:
Oh I don't think I'd twigged you were on alpine / busybox in here. Presumed it was a debian container with bash installed. Leave it as is.